### DISCUSSION TITLE
《黑客与画家》读书笔记 - Tw93

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#99] [General]
http://tw93.com/2018-07-08/hacker.html 

 这里是Tw93的技术博客，用于我的学习总结和研究，主要方向是前端，热爱分享、开源，最近在使用react和weex。Life最近在看《黑客与画家》这一本书，主要介绍黑客即优秀程序员的爱好和动机，讨论黑客成长、黑客对世界的贡献以及编程语言和黑客工作方法等所有对计算机时代感兴趣的人的一些话题，本文用于记录读书过程中的一些笔记和思考。

**Comment by tw93** [comment]
🏳️‍🌈🇺🇳🇦🇱🇪🇬🇦🇿🇦🇿🇦🇿🇴🇲🇦🇼🇦🇪🇦🇷🇦🇫🇦🇱🇺🇳🏳️‍🌈🚩🏁🇺🇳🇦🇱🇩🇿🇩🇿🇩🇿🇩🇿

**Comment by ghost** [comment]
学习了，谢谢博主！
